docs/index.bs

```diff
@@ -2973,16 +2984,48 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
               1. For each |eventType| of |settingsObject|'s [=environment settings object/global object=]'s associated list of <a>event listeners</a>' event types:
                   1. [=set/Append=] |eventType| to |workerGlobalScope|'s associated [=ServiceWorkerGlobalScope/service worker=]'s <a>set of event types to handle</a>.
 
-                  Note: If the global object's associated list of event listeners does not have any event listener added at this moment, the service worker's set of event types to handle remains an empty set. The user agents are encouraged to show a warning that the event listeners must be added on the very first evaluation of the worker script.
+                  Note: If the global object's associated list of event listeners does not have any event listener added at this moment, the service worker's <a>set of event types to handle</a> remains an empty set.
 
               1. Set |script|'s <a>has ever been evaluated flag</a>.
+              1. If the [=Empty Handler identification=] algorithm with |workerGlobalScope| returns true, the [=all fetch listeners are empty flag=] is set.
```

Comment:
I think you are talking about [Approaches to deal with the handler updates after the initialization](https://github.com/yoshisatoyanagisawa/service-worker-skip-no-op-fetch-handler#approaches-to-deal-with-the-handler-updates-after-the-initialization) in the explainer.  This implements [Option 2](https://github.com/yoshisatoyanagisawa/service-worker-skip-no-op-fetch-handler#option-2-updates-are-checked-every-time-when-the-service-worker-starts).  However, this just set "all fetch listeners are empty flag", and never unset.  Upon the algorithm, once the flag is set, it would be kept and does not behave like Option 2 (i.e. the fetch handler function body is evaluated every time service worker starts).  Let me fix.